### DISCUSSION TITLE
java ant open source project

### DISCUSSION THREAD

**Discussion by mahzad** [DependencyTrack/dependency-track#2426] [Q&A]
Is there any cyclonedx based tool for generating SBOM in **ant project with build.xml ?**

**Comment by stevespringett** [comment]
If you use Ant with Ivy, you should be able to use the Maven plugin with Ivy. If you're manually managing dependencies, then things become much harder.

**Reply by mahzad** [reply]
my project is EJBCA project . It seems that he did not use it. But I don't know what the exact technology is, can you help me?
this is github link:
**https://github.com/Keyfactor/ejbca-ce**
update: managing dependencies are manually . what is your recommendation? I used ODC 7.4 scan report but I like to track it with ODT

**Reply by stevespringett** [reply]
They already have a readme with all the components listed. Honestly, the easiest path forward would likely to contribute back a CycloneDX BOM with the same data, just in a machine-readable format. If that's not an option, then you could reach for a tool like [ORT](https://github.com/oss-review-toolkit/ort) which should be able to analyze those jars and output a proper BOM.
